en/file_formats/mission_plan.md

```diff
@@ -0,0 +1,168 @@
+items | The list of mission item objects associated with the mission. Each item will either be a [SimpleItem](#SimpleItem) or [ComplexItem](#ComplexItem). <!-- can you mix them? -->
+
+### Mission Item Object (SimpleItem) {#SimpleItem}
+
+A simple item represents a [MISSION_ITEM](../messages/common.md#MISSION_ITEM) command.
+
+```
+{
+    "autoContinue": true,
+    "command": 22,
+    "frame": 2,
+    "params": [
+        0,
+        0,
+        0,
+        0,
+        47.633127690000002,
+        -122.08867133,
+        50
+    ],
+    "type": "SimpleItem"
+},
+```
+
+The values in a `SimpleItem` map directly to the values in [MISSION_ITEM](../messages/common.md#MISSION_ITEM):
+
+Key | Description
+--- | ---
+type | `SimpleItem` (or `ComplexItem` for [complex items](#ComplexItem))
+autoContinue | [MISSION_ITEM](../messages/common.md#MISSION_ITEM).autoContinue
+command | [MISSION_ITEM](../messages/common.md#MISSION_ITEM).command
+frame | [MISSION_ITEM](../messages/common.md#MISSION_ITEM).frame
+params | [MISSION_ITEM](../messages/common.md#MISSION_ITEM).param1,2,3,4,x,y,z
+
+
+#### Special handling for DO_JUMP mission item
+
+Since `DO_JUMP` command requires you to specify the sequence number to jump to and the mission file format does not specify sequence numbers it requires special handling.
+
+First you must assign a unique identifier to the mission item you want to jump to:
+
+```
+{
+    ...
+    "doJumpId": 100
+}
+```
+
+The `doJumpId` can be any value greater than 0 and must uniquely identify a `DO_JUMP `target.
+
+Then in the actual `DO_JUMP` mission item you reference this unique id in the `SimpleItem.params` "param1" value (corresponding to `MISSION_ITEM.param1`):
+
+```
+{
+    ...
+    "command": 177,
+    "params": [
+        100,
+        ...
+    ],
+    ...
+},
+```
+
+When the mission is loaded the actual `DO_JUMP` sequence number will be determined and filled in.
+
+
+### Mission Item Object (ComplexItem) {#ComplexItem}
+
+A complex item is a higher level encapsulation of multiple [MISSION_ITEM](../messages/common.md#MISSION_ITEM) treated as a single entity.
+
+```
+{
+    "complexItemType": "survey",
+    "type": "ComplexItem",
+    "version": 3,
+    ...
+},
+```
+Complex items have two additional values associated with them:
+
+Key | Description
+--- | ---
+complexItemType | Specifies the type of complex item. QGroundControl currently supports the following types: [survey] <!-- (../file_formats/survey.md) -->, fwLandingPattern
+version | Specifies the version for this complex item.
+
+<!-- What of this is "standard", and what is PX4 stuff? -->
+<!-- Is survey format part of the standard?? -->
+
+
+## GeoFence Object {#geoFence}
+
+The (optional) geofence information associated with this flight plan has the same format as defined in [GeoFence Plan](../file_formats/geofence.md)
```

Comment:
Is this and Rally point doc correct - ie you could copy-paste in the geofence from the other doc, remove the file specific stuff and it would just "work"?

Note: My test of export from QGC left this list empty.

Reply:
The GeoFence and Rally Point file formats docs are both completely out of date.